aip/0166.md

```diff
@@ -0,0 +1,78 @@
+---
+aip:
+  id: 166
+  state: reviewing
+  created: 2020-06-24
+permalink: /166
+redirect_from:
+  - /0166
+---
+
+# Hierarchically-derived resources
+
+Services occasionally have resources where the resource's effective values are
+determined by consulting both configuration on the resource as well as
+configuration elsewhere, such as a parent in the hierarchy, and "flattening"
+them together. This is most common for configuration resources, which could be
+applied against a resource or a container.
+
+In these situations, users need to distinguish between properties applied to
```

Comment:
Why? I would argue that in most cases the user does not need to know if the portion of an effective resource/policy is directly bound or inherited. The application of an effective policy doesn't care how the effective policy was constructed.

The only use case I'm aware of is when someone is trying to audit a policy to determine where something is configured. (eg., answering "why do/don't I have access to this resource").

Reply:
> The only use case I'm aware of is when someone is trying to audit a policy to determine where something is configured. (eg., answering "why do/don't I have access to this resource").

That is the reason. :-)